src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -141,14 +143,6 @@ describe('getAuthenticatedAPIClient', () => {
   });
 });
 
-describe('logAPIErrorResponse', () => {
```

Comment:
Reviewers: FYI: Not sure what this was going to test, but it is a dup of the test above it and doesn't test api responses.  Note that I will be working in that area, so maybe I'll add a test then.